OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -243,31 +233,13 @@ public static bool Exists(string name)
 				return MountedFolders.Any(f => f.Exists(name));
 		}
 
-		static Dictionary<string, Assembly> assemblyCache = new Dictionary<string, Assembly>();
-
 		public static Assembly ResolveAssembly(object sender, ResolveEventArgs e)
 		{
 			foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
 				if (assembly.FullName == e.Name)
 					return assembly;
 
-			var frags = e.Name.Split(',');
```

Comment:
I'd like to work out what this is actually used for before we remove it.  I guess some docs-reading is in order.